Color Picker/ColorPickerScreen.swift

```diff
@@ -127,6 +127,126 @@ private struct BarView: View {
 	}
 }
 
+struct ColorInputView: View {
+    @EnvironmentObject private var appState: AppState
+    @State private var textColor: Color = .primary
+    @Binding var inputColorText: String
+    @Binding var isTextFieldFocused: Bool
+    @Binding var isPreventingUpdate: Bool
+    let colorPanel: NSColorPanel
+    let textFieldFontSize: Double
+    let inputColorType: colorType
+    enum colorType: String, CaseIterable {
+        case hex = "Hex"
+        case hsl = "HSL"
+        case rgb = "RGB"
+        case lch = "LCH"
+    }
+    var colorKeyboardShortcut: KeyEquivalent {
```

Comment:
This should be on the `ColorFormat` enum

Reply:
I've moved keyboardShortcut into an extension of ColorFormat within ColorPickerScreen. 

Would it be better to centralise this extension in Constants as not to have class extensions in different files ? or is it better to keep here, in order to limit access control of extension to this file only ?